rna-germline-variant-calling.wdl

```diff
@@ -494,16 +499,22 @@ task HaplotypeCaller {
   	File ref_fasta
   	File ref_fasta_index
 
+  	File dbSNP_vcf
+  	File dbSNP_vcf_index
+
   	Int preemptible_count
 
+  	Int? stand_call_conf
```

Comment:
I don't know that there's a standard way to do this, but in the prod WDL I've seen this written as:
Int stand_call_conf = 20
(and then down below it's just "-stand_call_conf ${stand_call_conf}")

Reply:
Also, I think the default is Q30.  Maybe ask @vdauwera or Laura?

Reply:
@eitanbanks if it makes most sense to use the built-in GATK default conf threshold, then I can change the WDL syntax to include `--stand_call_conf` argument only if a user has explicitly provide their own min threshold, and otherwise this argument can be omitted from the command line and utilize the GATK default without explicitly exposing it.

Reply:
yeah, I think that's best

Reply:
@eitanbanks GATK HaplotypeCaller docs seem to say that 20 should be the default for RNASeq https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.php

Reply:
@eitanbanks about the optional syntax, I'm not sure I can use the syntax you suggested as the `stand_call_conf` variable is an optional Int. Here's an example of some optional syntax from the prod pipeline handling an optional Float: https://github.com/broadinstitute/dsde-pipelines/blob/develop/genomes_in_the_cloud/single_sample/PairedSingleSampleWf.wdl#L1467

Reply:
Check out the read_length variable in the same wdl.
But I don't really care either way.  Was just adding a thought.